libbeat/docs/outputconfig.asciidoc

```diff
@@ -20,7 +20,7 @@ endif::[]
 
 ifndef::only-elasticsearch[]
 You configure {beatname_uc} to write to a specific output by setting options
-in the `output` section of the +{beatname_lc}.yml+ config file. Only a single
+in the `Outputs` section of the +{beatname_lc}.yml+ config file. Only a single
```

Comment:
We used `output` here to make it parallel with `output.elasticsearch` but not specific to one type of output. If you want to change this to Outputs to match the section comment, you can do so, but please remove the back ticks.